examples/pytorch/graphsage/dist/train_dist.py

```diff
@@ -307,8 +309,11 @@ def main(args):
     parser.add_argument('--ip_config', type=str, help='The file for IP configuration')
     parser.add_argument('--part_config', type=str, help='The path to the partition config file')
     parser.add_argument('--num_clients', type=int, help='The number of clients')
-    parser.add_argument('--n_classes', type=int, help='the number of classes')
-    parser.add_argument('--backend', type=str, default='gloo', help='pytorch distributed backend')
+    parser.add_argument('--n_classes', type=int, default=-1,
+                        help='the number of classes. If not specified, this value will be calculated'
```

Comment:
Capitalize the T.
and let's aim for 80 for the max line length.

Reply:
`80` seems a bit small. in this case, many blanks occupies in each line, so we will have many lines for this help message?

Reply:
Yes, it is fine to have multiple lines if the code is nicely laid out.